### PR TITLE
Fix/fortran gbl mdim

### PR DESCRIPTION
**Pull request by Ashutosh-Londhe**

### CODE REVIEW

**Comment by gihanmudalige** [comment]
@Ashutosh-Londhe The LB code should not be in this branch.  Actually it should not be committed to the repo at this time at all. Can you remove it and we work on it privately.

**Comment by Ashutosh-Londhe** [comment]
> @Ashutosh-Londhe The LB code should not be in this branch. Actually it should not be committed to the repo at this time at all. Can you remove it and we work on it privately.

sure, done

File: makefiles/Makefile.common [hunk 1]
```diff
@@ -30,7 +30,14 @@ OPS_LIB_OMPOFFLOAD=-lops_ompoffload
 OPS_LINK=
 CUDART=-lcudart
 
-include $(OPS_INSTALL_PATH)/../makefiles/Makefile.$(OPS_COMPILER)
+COMPILER_AVAILABLE := clang gnu intel icx intel_win mic pgi cray xl hip xlp8 aocc
```
Comment (reguly) [inline makefiles/Makefile.common:33]: this doesn't seem very scalable - could we just check what makefiles exist and throw the error if it doesn't?

**Review by reguly — APPROVED** [review]

File: makefiles/Makefile.common (hunk 1 above)
Comment (Ashutosh-Londhe) [reply makefiles/Makefile.common:33]: Sure, update the logic to check file exist or not

**Pull request merged**